server/src/test/integration/org/apache/cassandra/testing/CassandraTestTemplate.java

```diff
@@ -311,7 +309,7 @@ public Object resolveParameter(ParameterContext parameterContext, ExtensionConte
         }
     }
 
-    public static UpgradeableCluster retriableStartCluster(UpgradeableCluster.Builder builder, int maxAttempts)
+    public static UpgradeableCluster retryableStartCluster(UpgradeableCluster.Builder builder, int maxAttempts)
```

Comment:
this is a typo I fixed

Reply:
hmm.. I think it is not. see [retriable](https://en.wiktionary.org/wiki/retriable#:~:text=%2B%E2%80%8E%20%2Dable.-,Adjective,Capable%20of%20being%20retried.)
When adding suffix to words end with "y", "y" is changed to "i". See the [spell rule](https://www.languagecouncils.sg/goodenglish/resources/spelling-tips/adding-endings-to-words-that-end-in--y#:~:text=When%20adding%20endings%20to%20words,pretty%3A%20prettier%20%2F%20prettiest)

Reply:
it seems both are valid spellings . I will revert this change